UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -133,7 +133,28 @@ <h2>Introduction</h2>
 			<p>The good news is more and more publishers are creating publications that are Born Accessible (i.e.,
 				accessible from the outset, not fixed later) and getting the accessibility validation or audit done by
 				independent organizations.</p>
-		</section>
+		
+<section id="terminology">
+   <h3>Terminology</h3>
+   
+<p>There are several terms used in these guidelines that should   be defined  for clarity:</p>
+   <dl>
+      <dt><dfn>digital publication</dfn></dt>
+           <dd>
+         <p>A digital publication is a term that encompasses a wide range  of products  with a wide range of names. some examples are e-Books, audio books, manga, comic books, EPUB titles, PDF publications, journals, digital textbooks, picture books, children's picture books with accompanying audio, Digital Talking Books (DTB), and more.</p>
```

Comment:
While you have a good list of examples, I'm worried the actual definition is relying on the meaning of the term being self-evident. I think it might be better to give a bit more context, perhaps like:

```suggestion
         <p>The term digital publication is used in this document to refer to publications produced in any number of digital formats. Digital publications are not limited to books, but encompass any written, visual, or audio work distributed and read in digital form.</p>
         <p>Some examples of digital publications include ebooks, audiobooks, manga, comic books, journals, digital textbooks, picture books, and children's picture books with accompanying audio. The formats they come in include EPUB, PDF, and Digital Talking Books (DTB).</p>
```